introduction.md

```diff
@@ -0,0 +1 @@
+Itroduction to dojo as RIcky
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.